packages/xo-server/src/xo-mixins/logs/loggers/leveldb.js

```diff
@@ -23,6 +23,7 @@ export default class LevelDbLogger extends AbstractLogger {
 
     this._db = db
     this._namespace = namespace
+    this.getAll = db.getAll
```

Comment:
What's this?

Reply:
First: it should be bound or it will not work!

But it would be best to define this as a normal method.

Reply:
But this will not work because of the namespace.